tests/src/actionContainers/Swift3ActionContainerTests.scala

```diff
@@ -86,4 +86,34 @@ class Swift3ActionContainerTests extends SwiftActionContainerTests {
         err.trim shouldBe empty
     }
 
+    it should "ensure EDGE_HOST is available as an environment variable" in {
```

Comment:
I think this test is identical to the one in SwiftActionContainerTests - if it is, and since this suite extends the other, it will already run for swift:3 runtime.